components/Buttons/docs/README.md

```diff
@@ -35,3 +35,15 @@ elevation, Material Design ripples, and other stateful design APIs.
 - [Theming](theming.md)
 - [Color Theming](color-theming.md)
 - [Typography Theming](typography-theming.md)
+
+
+## Accessibility
+
+In order to get an accessible `MDCButton` make sure to set the accessibilty
+label. This is more important for `MDCFloatingButton` objects because they often
```

Comment:
I think we should be cautious about referring to something as `an accessible X`. Instead, maybe we can introduce a set of recommendations unique to Buttons. Setting an `accessibilityLabel` is not typical for buttons (since they usually have a title), but as @jverkoey points out, floating buttons may often omit their title.

As an alternative, perhaps we can set-up this section as an introduction to a checklist?

```markdown
To help ensure your buttons are accessible to as many users as possible, please be sure to review the following recommendations:

*  Set an appropriate [`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) value if your button does not have a title. This is often the case with `MDCFloatingButton` instances.
```